generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx.ejs

```diff
@@ -73,6 +75,13 @@ export class UserManagement extends React.Component<IUserManagementProps, IPagin
     this.props.getUsers(activePage - 1, itemsPerPage, `${sort},${order}`);
   }
 
+  toggleActive = ({ user }) => () => {
```

Comment:
unnecessary destructing, pass the user directly
`toggleActive = user => () => {..}` and `onClick={this.toggleActive(user) `